async-openai/src/types/run.rs

```diff
@@ -228,7 +228,8 @@ pub struct CreateRunRequest {
     pub tool_choice: Option<AssistantsApiToolChoiceOption>,
 
     /// Whether to enable [parallel function calling](https://platform.openai.com/docs/guides/function-calling/parallel-function-calling) during tool use.
-    pub parallel_tool_calls: Option<bool>,
+    #[serde(default)]
+    pub parallel_tool_calls: bool,
```

Comment:
This may work, however it has a semantics of a "required" param, which is not the source of truth - OpenAI API reference and spec says - its not a required parameter. 

I think core of the issue is not having `#[serde(skip_serializing_if = "Option::is_none")]` on all the `Option`-al  fields of `CreateRunRequest`  - if this attribute fixes the issues its more inline with semantics and consistency.